### ISSUE TITLE
Send sample email

### ISSUE THREAD

**Issue by digininja** [Raikia/FiercePhish#71]
I've just created an email template, it would be good to be able to have the option to send a copy of it to myself to test it. I know I can see a rendering of it in the page but seeing it in a mail client would be better. At the moment I need to create a list with just me in it and do a whole campaign.

**Comment by Raikia** [comment]
Yeah, this is something I found myself doing constantly and I have in v2.0 already 👍 

The short-fix is to click "Source" on the email template, copy it all, go to "Simple send", click "Source" on the text box, paste, click "Source" again.  Send.

But yeah, it needs a way to test the email.

**Comment by digininja** [comment]
The functionality of Eventbrite for sending emails is quite good, if you
can get access to an account then have a look at it. It is similar to yours
but you might get some ideas from it.

**Comment by Raikia** [comment]
Any chance you have a screenshot as an example?

**Comment by digininja** [comment]
I've just had another look and I think the only feature you are missing is
the test email option. I was thinking there was more as they bundle all the
timing stuff into the email page but you have it in the campaign section
which makes more sense for you.
